### ISSUE TITLE
Home Assistant integration

### ISSUE THREAD

**Issue by oicydwa** [TrackMyFish/TrackMyFish#14]
Pretty sure a lot of people that self host use some kind of home automation, and I know one of the biggest (if not THE biggest is home-assistant). I would like to see this have api configurations to send and receive notifications directly from home-assistant.

- Push notifications for feeding reminders and water changes. 
- If TMF has an inventory tracker (another suggested addition), can integrate a list in Hass to show stock levels of chemicals and food stuffs
- Reminders for sick fish treatment and quarantine tracking (another suggested addition) to Hass

**Comment by simondrake** [comment]
I don't personally use home-assistant, but this sounds like a great idea. 

> If TMF has an inventory tracker (another suggested addition)

> Reminders for sick fish treatment and quarantine tracking (another suggested addition)

Could you raise these two as new/separate requests please @oicydwa, for easier tracking?

**Comment by simondrake** [comment]
Note for future readers (as I didn't know): [HASS](https://www.home-assistant.io/docs/tools/hass/) is the home-assistant command-line tool.

**Comment by oicydwa** [comment]
Correct. Home assistant has a very robust API system. It does work through the backend, obviously, but inthis case, I wasn't referring directly to that.

**Comment by simondrake** [comment]
Makes sense. I've marked this as "help wanted" as I don't have much experience with HA. 

Otherwise I'll look into it once I have time 🙂

**Comment by oicydwa** [comment]
Not a problem. I have never worked with their API, but I have quite a bit of experience with it. I'll see what I can figure out and make this a reality. Might take some time though.

**Comment by simondrake** [comment]
No rush 🙂 

Feel free to reach out if you need any help.
